components/Navigation.js

```diff
@@ -28,10 +29,14 @@ const NavItem = styled(Link)`
 `;
 
 export default function Navigation() {
+  const router = useRouter();
+  const pathname = router.pathname;
+  pathname.includes("/");
+
   return (
     <NavbarContainer>
       <NavItem href="/">Home</NavItem>
-      <NavItem href="/create">New Card</NavItem>
+      <NavItem href={"/create-card"}>+</NavItem>
```

Comment:
Right now, the ”+” button has no context for screen readers. I suggest to add an `aria-label`:

```suggestion
      <NavItem href="/create-card" aria-label="Create a new flashcard">+</NavItem>
```